### DISCUSSION TITLE
How to see the latest changes in a project?

### DISCUSSION THREAD

**Discussion by volkan0x** [zed-industries/zed#12195] [Feature Requests]
Is there any way to see the change history of a file or the project on Zed? You can easily do this in Pycharm like this:
<img width="446" alt="Screenshot 2024-05-23 at 14 45 32" src="https://github.com/zed-industries/zed/assets/68243739/d9d9682f-517c-40e3-9382-28865037b5d1">

**Comment by SomeoneToIgnore** [comment]
No way currently, unfortunately.

There's a tracking issue that will enable something similar via git: https://github.com/zed-industries/zed/issues/8665 but we're not there yet.

As for the local history, it is a superb feature, but not planned currently and is far-far away from being introduced: current idea is to work on a CRDB first, that will automagically enable this feature when integrated into Zed.
I have no other details about this project except that it's not actively worked on.

**Comment by chrisveness** [comment]
I'm a WebStorm user (on Linux) hoping to move to Zed. I've found WebStorm's 'Local History' an occasional lifesaver, and dismayed it's not available in Zed.

I love Zed, but I'm not sure if I'm ready to move to it without that safety-net being available.

I hope this can be pushed up the priorities.

Fantastic work on Zed, I love it!
